### PR TITLE
Add basic logging configurations

### PR DESCRIPTION
**Pull request by tzwang1**
Add logging configurations with different log levels for local development and production.

### CODE REVIEW

File: dietstory-routing/registration/views.py [hunk 1]
```diff
@@ -124,24 +134,30 @@ def post(self, request, *args, **kwargs):
...
 
                     try:
-                        send_verification_email(account.email, account_activation_token.make_token(account))
+                        token = account_activation_token.make_token(account)
+                        logger.debug("Verification Token: {}".format(token))
+                        send_verification_email(account.email, token)
                     except IOError:
-                        print("Failed to send email.") #TODO: Logging
-
+                        logger.warn("[RV-1] Failed to send email to: {}".format(account.email))
```
Comment (albscui) [inline dietstory-routing/registration/views.py]: Why is this a warning when it is under an IOError?

File: dietstory-routing/registration/views.py [hunk 2]
```diff
@@ -223,22 +239,23 @@ def post(self, request, *args, **kwargs):
         try:
             user = Accounts.objects.get(email=email)
         except (TypeError, ValueError, OverflowError, Accounts.DoesNotExist):
+            logger.warn("[RV-4] Email: {} does not exist.".format(email))
```
Comment (BenjixD) [inline dietstory-routing/registration/views.py]: This is actually totally normal - a user may submit a wrong/non-existent email

File: dietstory-routing/registration/views.py (hunk 1 above)
Comment (BenjixD) [reply dietstory-routing/registration/views.py]: Account is actually created despite email failing to send verification token.

Comment (albscui) [reply dietstory-routing/registration/views.py]: Hmm I see, if this is for sending the user the verification token, and we failed to send them a verification token, then isn't their account unuseable? Do we have a mechanism for resending or dealing with this issue?

Comment (BenjixD) [reply dietstory-routing/registration/views.py]: Yes - I believe a resend-verification token was created but not wired on the client side yet. So it is possible for resending.

The problem is that we depend on a 3rd party email service, and our email can block after a limited amount of sends. We recognize this issue but don't necessarily have a way to solve it yet.

File: dietstory-routing/registration/views.py [hunk 3]
```diff
@@ -587,7 +620,7 @@ def post(self, request, *args, **kwargs):
             400:
                 content:
                     application/json:
-                        schema:
+                        schema::
```
Comment (BenjixD) [inline dietstory-routing/registration/views.py]: Typo - yaml won't parse properly.

File: dietstory-routing/dietstory-routing/settings.py [hunk 4]
```diff
@@ -77,6 +77,26 @@
...
+        },
+    },
+    'loggers': {
+        'default-logger': {
+            'handlers': ['console'],
+            'level': os.getenv('DJANGO_LOG_LEVEL', 'INFO'),
+        },
+        'dev-logger': {
+            'handlers': ['console'],
+            'level': os.getenv('DJANGO_LOG_LEVEL', 'DEBUG')
```
Comment (BenjixD) [inline dietstory-routing/dietstory-routing/settings.py]: We should move the log level setting (ie `os.getenv('DJANGO_LOG_LEVEL', 'DEBUG')`) to the config file.

Comment (BenjixD) [reply dietstory-routing/dietstory-routing/settings.py]: Make sure to update the secret_template.json when introducing the new setting.

Comment (tzwang1) [reply dietstory-routing/dietstory-routing/settings.py]: Moved to config file and updated the template.

**Pull request merged**